test_schemas/en/test_list_collector_two_list_collectors.json

```diff
@@ -0,0 +1,443 @@
+        {
+            "id": "section",
+            "title": "Household members",
+            "summary": {
+                "show_on_completion": true,
+                "items": [
+                    {
+                        "type": "List",
+                        "for_list": "people",
+                        "title": "Household members",
+                        "add_link_text": "Add someone to this household",
+                        "empty_list_text": "There are no householders"
+                    }
+                ]
+            },
+            "groups": [
+                {
+                    "id": "group",
+                    "title": "People",
+                    "blocks": [
+                        {
+                            "id": "anyone-usually-live-at",
+                            "question": {
+                                "answers": [
+                                    {
+                                        "id": "anyone-usually-live-at-answer",
+                                        "mandatory": true,
+                                        "options": [
+                                            {
+                                                "action": {
+                                                    "params": {
+                                                        "block_id": "add-person",
+                                                        "list_name": "people"
+                                                    },
+                                                    "type": "RedirectToListAddBlock"
+                                                },
+                                                "label": "Yes",
+                                                "value": "Yes"
+                                            },
+                                            {
+                                                "description": "For example, this is a second address or holiday home",
+                                                "label": "No",
+                                                "value": "No"
+                                            }
+                                        ],
+                                        "type": "Radio"
+                                    }
+                                ],
+                                "id": "anyone-usually-live-at-question",
+                                "title": "Does anyone else live at your address?",
+                                "type": "General"
+                            },
+                            "routing_rules": [
+                                {
+                                    "goto": {
+                                        "block": "list-collector"
+                                    }
+                                }
+                            ],
+                            "type": "Question"
+                        },
+                        {
```

Comment:
This list collector needs a driving question, and it needs to be answered in the negative to make sure the list collector is not on the path